Workflows/Architecture review.md

```diff
@@ -26,9 +26,30 @@ Please follow this timeline:
 8. Once you've finished the database design, it has to be approved by someone from the management.
 9. Once your architecture has been approved, you can generate a new Rails project. Read more about that on the next page.
 
+Ask the project manager to give you the project specification.
+
+## Database design & discuss major implementation details
+Once you're done with the initial project meeting and have the specification, you can start designing the database. Use an online tool so you can easily present the diagram later on the review, such as [dbdiagram](https://dbdiagram.io/home) or [SqlDBM](https://sqldbm.com/Home/), or some other.
+
+Consult the project manager at anytime and add new discoveries to the specification if you find that something's missing. Don't hesitate to involve other team members since this is the most important part of the app.
+
+When you're done with the database diagram, sit down with the member of your team that was at the meeting with you and go through:
+* the database diagram
+* authentication
+ - what gems will be used
+ - API authentication - cookies (web frontend) or jwt/custom tokens (mobile frontend), or both
+* admin UI setup
+* emails - templates (preferred) or custom design (rarely a good use of time for MVP)
+* any 3rd party services
+* any other project specific problems
+
+## The review meeting
+Once you've finished the database design and discussed other major implementation details with the team member, it has to be approved by the project sponsor. The next step is to ping the project sponsor to organise the review meeting. At the meeting the main focus will be on the database design. Standard implementation details will be covered briefly. This meeting is a good place to dicuss any other uncertainties or problems.
```

Comment:
I'm not sure of the sponsor role here, shouldn't the technical part be signed of by the system/solutions architect?

Reply:
@d4be4st what do you think?